iotdb-core/datanode/src/main/java/org/apache/iotdb/db/schemaengine/table/DataNodeTableCache.java

```diff
@@ -99,6 +100,11 @@ public void init(final byte[] tableInitializationBytes) {
               databaseTableMap.put(
                   PathUtils.unQualifyDatabaseName(key),
                   value.stream()
+                      .filter(
```

Comment:
use filter to do for-each style is not a good habit. You can do another for-each out of the databaseTableMap.put